VMCreate.yml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
perhaps we should have comment here to describe what the playbook does in more details

Reply:
update